### PR TITLE
reformat setup of commandline scripts in package

### PR DESCRIPTION
**Pull request by teslajoy**
please don't merge this pull request until I have assigned a reviewer. Thanks :)

### CODE REVIEW

File: annotator/__main__.py [hunk 1]
```diff
@@ -0,0 +1,440 @@
...
+    :param tableSynId:
+    :param newTable:
+    :param releaseVersion:
+    :return:
+    """
+    currentTable = syn.tableQuery("SELECT * FROM %s" % tableSynId)
+
+    # If current table has rows, delete all the rows
+    if currentTable.asRowSet().rows:
+        deletedRows = syn.delete(currentTable.asRowSet())
```
Comment (sgosline) [inline annotator/__main__.py]: Do we want to delete all annotations? Or just update? Should there be an option to do both?

File: annotator/__main__.py [hunk 2]
```diff
@@ -0,0 +1,440 @@
...
+
+    :return:
+    """
+    parser = argparse.ArgumentParser()
+
+    subparsers = parser.add_subparsers(title='commands',
+                                       description='The following commands are available:',
+                                       help='For additional help: "annotator <COMMAND> -h"')
+
+    parser_meltjson = subparsers.add_parser('meltjson', help='Creates a flattened synapse table from json files '
```
Comment (sgosline) [inline annotator/__main__.py]: `meltjson` is a bit of a weird term for those of us not in the tidyverse. perhaps use `mergejson` or `combinejson` as an argument?

File: annotator/__main__.py (hunk 1 above)
Comment (teslajoy) [reply annotator/__main__.py]: I had an update function, but in this case @kdaily suggested to remove the complexity of updating a table and just delete the entire table. It happens to be faster too :)

File: annotator/__main__.py (hunk 2 above)
Comment (teslajoy) [reply annotator/__main__.py]: how about json2table? we are doing one more step (un-nesting) after merging/combining.

Comment (teslajoy) [reply annotator/__main__.py]: or something more creative... need some sara help here.

Comment (sgosline) [reply annotator/__main__.py]: json2table!

**Pull request merged**